wordpress-mobile/WordPress-Android
# [Gutenberg] Voice to content

**Related PRs:**
- https://github.com/wordpress-mobile/gutenberg-mobile/pull/6878
- https://github.com/WordPress/gutenberg/pull/61796

-----

## To Test:

A [hard-coded call](https://github.com/wordpress-mobile/WordPress-Android/pull/20852/files/e0145b44648e392051644437db7e87345918e8f9#diff-3d055f27ee69cf53a3055662daab1fc04df6e0eba7deadfd63c3a79df8e96dc5) has been added to test the integration with the editor.

- Install the build from this RP
- Open the editor
- Expect the content to be added once the editor opens.

-----

## Regression Notes

1. Potential unintended areas of impact

    - Editor

2. What I did to test those areas of impact (or what existing automated tests I relied on)

    - Manual testing

3. What automated tests I added (or what prevented me from doing so)

    - Integration tests were added in Gutenberg Mobile

-----

## PR Submission Checklist:


-----

## Testing Checklist (strike-out the not-applying and unnecessary ones):
